src/components/aeBanner/aeBanner.scss

```diff
@@ -0,0 +1,31 @@
+@import "../variables.scss";
+@import "../mixins.scss";
+
+.ae-banner {
+  background: linear-gradient(to bottom, rgba($aubergine, 0.65), transparent);
+  z-index: 1;
+  padding: 10px 10px 25px 10px;
+
+  > div {
+    display: flex;
+    align-items: center;
+    height: 64px;
```

Comment:
I think at best having a padding instead of a fixed height would be ok. 
`
padding: 17px 22px;
`

Reply:
i agree